src/plugins/settings.rs

```diff
@@ -178,6 +180,31 @@ impl SettingsPlugin {
                         });
                     });
                 });
+                if data.dev_mode {
```

Comment:
This will make it so console output is enabled by default, and to disable it we have to enable dev mode first.
We can just always show this option